File: test/ocamlTests/IntTest/negateTest.ml
test "negate(8)" (fun () -> expect (Int.negate 8) |> toEqual Eq.int -8) ; 
test "negate(-7)" (fun () -> expect (Int.negate -7) |> toEqual Eq.int 7) ; 
test "negate(0)" (fun () -> expect (Int.negate 0) |> toEqual Eq.int 0) ; 

File: test/fsharpTests/IntTest/negateTest.fs
testCase "negate(8)" 
<| fun _ -> 
    let expected = -8
    Expect.equal expected (Int.negate 8) "error"
testCase "negate(-7)" 
<| fun _ -> 
    let expected = 7
    Expect.equal expected (Int.negate -7) "error"
testCase "negate(0)" 
<| fun _ -> 
    let expected = 0
    Expect.equal expected (Int.negate 0) "error"
